app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
+
+/*navbar*/
+
+#headWrapper{
+  height: 60px;
+  border-bottom: 2px solid;
+  margin-bottom:30px;
+  background-color:#373d46;
+  color: #4cc5e2;
+}
+
+#headWrapper header {
+  height: 60px;
+  width: 960px;
+  margin: auto;
```

Comment:
It is better to have a separate class with these two css rules (width &  margin), and just add the class to each element you want to behave this way.
Two benefits you get from this:
1. you write less code => smaller files
2. you can maintain the width form one place in case you want to change the width